packages/taxdat/R/snap_to_time_period.R

```diff
@@ -188,7 +188,7 @@ drop_multiyear <- function(df,
     cat("---- Dropping", length(ind_diff), "observations that span multiple years from admin levels",
         paste(admin_levels, collapse = "-"), ":\n")
     
-    print(df[ind_diff, ] %>% sf::st_drop_geometry())
```

Comment:
Why are we removing this drop geometry?

Reply:
To avoid all the output of geometries in the print statement.